src/ch02/TrafficLight.enum.ts

```diff
@@ -1,5 +1,42 @@
-export enum TrafficLight {
+export enum RawTrafficLight {
     RED,
     YELLOW,
     GREEN
 }
+class Car {
+    stop(): boolean {
+      return false;
+    }
+  
+    drive(): boolean {
+      return true;
+    }
+  }
+const car = new Car();
+export interface TrafficLight{
```

Comment:
```suggestion
  }
 
const car = new Car();

export interface TrafficLight{
```
이 해당 문단은 띄어쓰기 필요합니다.